cmd/glasskube/cmd/bootstrap.go

```diff
@@ -90,13 +91,25 @@ var bootstrapCmd = &cobra.Command{
 			fmt.Fprintf(os.Stderr, "\nAn error occurred during bootstrap:\n%v\n", err)
 			cliutils.ExitWithError()
 		}
-		if err := printBootsrap(
-			manifests,
-			bootstrapCmdOptions.Output,
-		); err != nil {
-			fmt.Fprintf(os.Stderr, "\nAn error occurred in printing : %v\n", err)
-			cliutils.ExitWithError()
+		if bootstrapCmdOptions.dryrun {
+			fmt.Println("Dry run completed successfully.")
```

Comment:
This prints the message "Dry run completed successfully." to `stdout`. If you run the bootstrap dryrun in combination with `--output yaml` (which is probably the main use case I think), this will put that exact string into the yaml, making it an invalid yaml. That's why such messages are always printed to `stderr` instead. 

Please test your solution like this: `glasskube bootstrap --dry-run --o yaml > output.yaml` and then check the content of the `output.yaml`. This must be valid yaml which could be used with `kubectl apply` for example (please test that this actually works).

Reply:
Ohk, so here the output comes something like this:
```
flux-system
glasskube-system
critical-pods
buckets.source.toolkit.fluxcd.io
gitrepositories.source.toolkit.fluxcd.io
helmcharts.source.toolkit.fluxcd.io
helmreleases.helm.toolkit.fluxcd.io
helmrepositories.source.toolkit.fluxcd.io
ocirepositories.source.toolkit.fluxcd.io
packageinfos.packages.glasskube.dev
packagerepositories.packages.glasskube.dev
packages.packages.glasskube.dev
helm-controller
source-controller
glasskube-controller-manager
glasskube-webhook-cert
glasskube-leader-election-role
glasskube-webhook-cert-role
crd-controller
flux-edit
flux-view
glasskube-metrics-reader
glasskube-packagerepository-editor-role
glasskube-packagerepository-viewer-role
glasskube-proxy-role
glasskube-webhook-cert-role
glasskube-leader-election-rolebinding
glasskube-webhook-cert-rolebinding
cluster-reconciler
crd-controller
glasskube-manager-rolebinding
glasskube-proxy-rolebinding
glasskube-webhook-cert-rolebinding
source-controller
glasskube-controller-manager-metrics-service
glasskube-webhook-service
helm-controller
source-controller
glasskube-controller-manager
glasskube-webhook-cert
glasskube-webhook-cert-init
allow-egress
allow-scraping
allow-webhooks
glasskube-validating-webhook-configuration
glasskube
Dry run completed successfully. The following manifests would be applied:
apiVersion: v1
kind: Namespace
metadata:
  name: flux-system
---
```

Here we don't require the texts above the `yaml`
The output should be something like this:
```
apiVersion: v1
kind: Namespace
metadata:
  name: flux-system

---
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    packages.glasskube.dev/telemetry-enabled: "true"
    packages.glasskube.dev/telemetry-id: b753e233-32ac-47eb-a25b-299b74094587
  labels:
    app.kubernetes.io/component: manager
    app.kubernetes.io/created-by: glasskube
    app.kubernetes.io/instance: system
    app.kubernetes.io/managed-by: kustomize
    app.kubernetes.io/name: namespace
    app.kubernetes.io/part-of: glasskube
    control-plane: controller-manager
  name: glasskube-system

---
```

Reply:
Yes, the output should only be the yaml that you can apply. So it should look like the second code snippet.